packages/backend/src/config/Config.ts

```diff
@@ -65,3 +66,8 @@ export interface DiscoveryConfig {
   readonly alchemyApiKey: string
   readonly etherscanApiKey: string
 }
+
+export interface MetricsConfig {
```

Comment:
i think it would be nice to have authentication-free version for local environments. This probably should print out warning in the console.